spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -600,7 +600,7 @@ def and_i_have_reordered_the_steps
       content_items: step_link_fixtures_content_items,
     )
     visit step_by_step_page_reorder_path(@step_by_step_page)
-    find(".js-reorder:first-child .js-down").click
+    find(".app-c-reorderable-list__item:first-child .js-reorderable-list-down").click
```

Comment:
Will the same approach we used for Coronavirus steps, `all("button", text: "Down").first.click`, work here?

Reply:
yep works